alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/MonitorRunnerIT.kt

```diff
@@ -582,6 +582,116 @@ class MonitorRunnerIT : AlertingRestTestCase() {
                 actionResults2[actionThrottleEnabled.id]!!.lastExecutionTime)
     }
 
+    fun `test monitor HttpInput with non JSON response `() {
+        val clusterIndex = randomInt(clusterHosts.size - 1)
+        val input = randomHttpInput(
+                scheme = clusterHosts[clusterIndex].schemeName,
+                host = clusterHosts[clusterIndex].hostName,
+                port = clusterHosts[clusterIndex].port,
+                path = "_cat/plugins",
```

Comment:
I changed the path, and the original one is https://github.com/opendistro-for-elasticsearch/alerting/pull/82/files#diff-6a0bc20035e8972567e050ee46d79e13R646